doc/samtools-merge.1

```diff
@@ -106,6 +106,19 @@ are ignored.)
 The input alignments are sorted by read names rather than by chromosomal
 coordinates
 .TP
+.BI -o \ FILE
```

Comment:
Perhaps the synopsis should include the new option, as well.

Reply:
I thought about it, both here and in the Usage display. It needs to be in both if this is to become the new “standard”, but conversely dual synopses kind of increases the confusion… Happy to add it if consensus develops…

It probably will need a NEWS entry too, so people can look back and see when this syntax became available.

Reply:
Maybe:
```
Usage: samtools merge -o <out.bam> [-nurlf] [-h inh.sam] [-b <bamlist.fofn>] <in1.bam> [<in2.bam> ... <inN.bam>]
or:    samtools merge [-nurlf] [-h inh.sam] [-b <bamlist.fofn>] <out.bam> <in1.bam> [<in2.bam> ... <inN.bam>]
```

Deliberately moving the output file to emphasise that it doesn't need to be with the list of inputs.

Reply:
A NEWS entry is definitely required.